### PR TITLE
🌹 Add blog URL to settings page

### LINKED ISSUE #437: Add URL to app to blog site

### PR DESCRIPTION
**Pull request by viktorgardart**
~Fixes #437~

Edit: Doesn't actually solve #437

### CODE REVIEW

**Comment by rnystrom** [comment]
Lovely!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
Oh ha, meant "add a url to the app on blog.githawk.com", but this is also nice! 😀

File: Classes/Settings/SettingsViewController.swift [hunk 1]
```diff
@@ -115,6 +118,12 @@ NewIssueTableViewControllerDelegate {
 		presentSafari(url: url)
     }
 
+    func onViewBlog() {
+        guard let url = URL(string: "http://blog.githawk.com/")
```
Comment (Sherlouk) [inline Classes/Settings/SettingsViewController.swift]: This is less a note with this PR but more a general statement, are we okay with generic strings like this being placed throughout the codebase? Or would we rather move them to either a constants file, or maybe the Info.plist under it's own dictionary? Easier to manage long-term?

Comment (viktorgardart) [reply Classes/Settings/SettingsViewController.swift]: This the best shot would be to store them in a constants file. And then call it by: `Constants.url.blog` or something similar

Comment (BasThomas) [reply Classes/Settings/SettingsViewController.swift]: We already have [Strings.swift](https://github.com/rnystrom/GitHawk/blob/master/Classes/Views/Strings.swift), maybe that works for now as well?

Comment (viktorgardart) [reply Classes/Settings/SettingsViewController.swift]: Created a new issue for this:

https://github.com/rnystrom/GitHawk/issues/498

Comment (BasThomas) [reply Classes/Settings/SettingsViewController.swift]: Awesome!

**Comment by rnystrom** [comment]
Dang, sorry bug I'm actually going to back out of this. The more I thought about it, it might end up just bloating the settings. I want to keep them as light as possible for now. Will revive if the need comes up!

**Comment by BasThomas** [comment]
Think at some point we should improve there. Think it can be better even at this point. Maybe declutter by having eg. the switches in a new vc, like accounts?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request closed without merging**